clearblade.md

```diff
@@ -1288,9 +1288,15 @@ Class: ClearBlade.Database();
 This class allows you to specify the query in raw SQL on platform and edge instead of using the existing ClearBlade query model. The function can be used for operations, such as `JOIN` and `SOME`, that are not supported by the ClearBlade Query Model.
 
 ~~~javascript
+//Issues commands for external collections
 		var db = ClearBlade.Database();
 ~~~
 
+~~~javascript
+//Issues commands for external database connections
```

Comment:
Similarly here, instead of the comment, write an explanation above the code snippet.

Use the Database object with the external DB options object for performing operations on external database connections.